_maps/pubbystation.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`Duplicate APC created at Engineering Maintenance (127,116,2)`
<hr>
For some reason, a second APC has been placed right next to the normal one and pixel shifted upwards so that it's hidden under the 'NO ERP' poster.

Right clicking on the tile and deleting the APC labelled 'north' fixes the error.

(To clarify, the bad APC is actually on the corner tile below the poster.)

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/57483089/169619577-87e5293d-02ae-4d79-b078-11aba56aea2b.gif)